wagtail/core/tests/test_page_permissions.py

```diff
@@ -307,6 +307,37 @@ def test_superuser_has_full_permissions(self):
         # cannot move because the parent_page_types rule of BusinessSubIndex forbids EventPage as a parent
         self.assertFalse(board_meetings_perms.can_move_to(unpublished_event_page))
 
+    def test_cant_move_pages_between_locales(self):
+        user = get_user_model().objects.get(username='superuser')
```

Comment:
This needs updating to work with our new EmailUser test run (#6330):
```suggestion
        user = get_user_model().objects.get(email='superuser@example.com')
```